release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -505,6 +505,21 @@ def split_input_files_per_worker(args):
     ]
 
 
+def get_s3fs_with_boto_creds():
```

Comment:
Can you create a GH issue, and point users to this workaround?
And can you add a comment here on why doing this, as well as the downsides of this workaround?
IIUC, the downsides are:
* credentials obtained from the driver may not work on remote nodes.
* this approach doesn't handle expiration.

Reply:
linked to this existing issue [here](https://github.com/ray-project/ray/issues/47230)